src/ol/source/GeoTIFF.js

```diff
@@ -298,6 +299,20 @@ function getMaxForDataType(array) {
   return 255;
 }
 
+/**
+ * @typedef {Object} GeoTIFFSourceOptions
+ * @property {boolean} [forceXHR=false] Whether to force the usage of the browsers XMLHttpRequest API.
+ * @property {Object} [headers] additional headers to be passed with each request.
```

Comment:
I assume this is an object literal of key-value pairs?
```suggestion
 * @property {Object<string, string>} [headers] additional key-value pairs of headers to be passed with each request. Key is the header name, value the header value.
```

Reply:
Yes, right! Will include this.